src/core/core.scale.js

```diff
@@ -1194,28 +1197,43 @@ export default class Scale extends Element {
 				continue;
 			}
 
-			alignedLineValue = _alignPixel(chart, lineValue, lineWidth);
-
-			if (isHorizontal) {
-				tx1 = tx2 = x1 = x2 = alignedLineValue;
-			} else {
-				ty1 = ty2 = y1 = y2 = alignedLineValue;
+			let lineDistance = 0;
+			if (gridLines.linesBetweenTicks !== 0) {
+				lineDistance = (lineValue - previousLineValue) / (gridLines.linesBetweenTicks + 1);
```

Comment:
What do you think about adding these into the `ticks` array when it is built? It would require ensuring the label does not show, but would mean that the extra config options wouldn't be needed. @kurkle thoughts?

Reply:
Sure, but this would mean that these lines cannot be configured on their own. I think it would be useful to have config options for these lines (at least the width, but dash and color would also be useful to add as well).

Reply:
@etimberg do you think this feature could be useful for ChartJS? and what would you like to have changed compared to this pull request? More configurations (width, dash, and color?)? And do you want to access this feature in a different way, for example adding it directly in the ticks array or in a callback? Then I could look at it again and implement it that way.

Reply:
@JoostBaars sorry for the delay. I was hoping @kurkle could provide some input before I requested changes.

At minimum, I think the configuration should include the same options as grid lines (width, dash, colour). In terms of accessing these values through the `tick` array, I'm torn. On one hand, I think from a modelling perspective they are better put there. The time scale already has the concept of major/minor ticks and there is configuration overrides for the major ticks. https://www.chartjs.org/docs/master/axes/styling#major-tick-configuration

V2 used to have a minor tick override, but it was removed as it turned out to not be very useful. Perhaps this could be leveraged to format these intermediate grid lines?

Reply:
@etimberg Alright, I'll add the other options as the gridlines. I will look more into the location where to access these values (the minor tick override of V2 or the tick array).

Reply:
Sorry, I've been busy the last time... I have added the other settings but did not look for a better place yet to store these parameters.